weaver/wps_restapi/swagger_definitions.py

```diff
@@ -495,6 +547,14 @@ class FormatDescription(FormatDefault, FormatExtra):
     )
 
 
+class DescribeFormatDescription(Format, FormatExtra, FormatDefault):
+    pass
+
+
+class DeployFormatDescription(FormatSelection, FormatExtra, FormatDefault):
+    pass
```

Comment:
I would like to have a quick note again here that explains the difference of dependencies. 
(`mediaType` only during describe following conversion, but retro-compatibility allowed during deploy with either old `mimeType` or new `mediaType`)